addons/default/visiosoft/base-theme/resources/views/addons/anomaly/posts-module/posts/articles.twig

```diff
@@ -12,12 +15,12 @@
                 <div class="px-2">
                     <div
                         class="fs-14 product rounded position-relative bg-white link-dark text-decoration-none border">
-                        <div class="product-img">
-                            <a href="{{ post.route('view') }}">
-                                {{ img( post.cover_image.make.url ?? 'visiosoft.theme.base::images/no-image.png').class('w-100')|raw }}
+                        <div class="product-img d-flex justify-content-center">
+                            <a href="{{ post.route('view') }}" class="">
+                                {{ img( post.cover_image.make.url ?? 'visiosoft.theme.base::images/no-image.png').class("w-600")|raw }}
```

Comment:
you can use font-weight-bold or font-weight-bolder instead of w-600

Reply:
font-weight-bold is equal to 700, and bolder is "bolder" than that. I guess w-600 has a purpose. should I change it anyway?